src/plugins/flexibleLayout/toolbarProvider.js

```diff
@@ -98,7 +99,7 @@ function ToolbarProvider(openmct) {
                   emphasis: 'true',
                   callback: function () {
                     openmct.objectViews.emit(
-                      'contextAction',
+                      `contextAction:${primaryKeyString}`,
```

Comment:
gah, nice catch

Reply:
So basically all the layouts were getting this event instead of the single one we wanted. It seems like it'd be good practice to have events names always have the domain key slapped on them just in case, unless you really intend it for a nested object (rare)

Reply:
Yeah. Having to handle all those events is costly. This might be a good solution for some cases